frontend/device/input.lua

```diff
@@ -240,6 +241,12 @@ function Input:adjustTouchTranslate(ev, by)
     end
 end
 
+function Input:adjustKindleOasisOrientation(ev)
```

Comment:
Perhaps moving this into `frontend/device/kindle/device.lua` to keep input.lua generic?